packages/components/src/molecules/ProductTitle/ProductTitle.tsx

```diff
@@ -26,9 +26,33 @@ export interface ProductTitleProps
    */
   refNumber?: string
   /**
-   * The current value of the rating, a number from 0 to 5.
+   * Object containing the rating value and reviews count
```

Comment:
🎗️  ask doc request when merging the review and rating feature.
(updates ProductTitle docs)